src/Query.cpp

```diff
@@ -1,9 +1,13 @@
 #include "Query.h"
 
+#include <QVariant>
+
 Query::Query(ConnectionManager* connectionManager, QObject *parent) :
     AbstractQuery(connectionManager, parent)
 {
-
+    m_numberTypeIds << 2 << 3 << 6 << 32 << 4 << 35 << 5 << 36 << 38;
```

Comment:
I'd use the named types instead of raw ids to make it clear which are included.

`m_numberTypeIds << QVariant::Type::Int ...`